src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  //for bg image to fill the whole screen
+  //background-position: center;
+  //background-size: cover;
+  background-color: lightblue;
```

Comment:
Instead of using a hardcoded color like 'lightblue', consider using an SCSS variable to make it easier to manage colors. For example, you can define a variable $headerBackgroundColor and use it here. 😊